cli/tools/compile.rs

```diff
@@ -365,16 +365,18 @@ fn get_module_roots_and_include_paths(
       | MediaType::Dts
       | MediaType::Dmts
       | MediaType::Dcts
-      | MediaType::Tsx
-      | MediaType::Json
-      | MediaType::Wasm => true,
+      | MediaType::Tsx => true,
       MediaType::Css
       | MediaType::Html
       | MediaType::Jsonc
       | MediaType::Json5
+      | MediaType::Json
```

Comment:
I don't have an issue with json files, but they should not be part of module graph unless being imported either.